sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1729,19 +1732,21 @@ def _flush_batch(self, destination):
     if destination in self._destination_buffer_byte_size:
       del self._destination_buffer_byte_size[destination]
 
```

Comment:
In the retry loop above, with

```
rows = [fr[0] for fr in failed_rows]
````
do we also need to update `window_values` so the indices line up? (Looking at this, shouldn't insert_ids have been updated as well?)